Azure/sonic-buildimage
# [device/celestica] Implement PSU APIs based on the new platform API

**- What I did**

- Implement part of the Chassis and Psu related APIs.
- Psu APi
  - get_fan()
  - set_status_led()

**- How I did it**
1. Implement API based on sonic platform API base
3. Init Psu list in Chassis class.

**- How to verify it**
- Update HLX psu driver (https://github.com/Azure/sonic-buildimage/pull/2871)
- Create python script follow the new platform API design doc to verify it (Azure/SONiC#285.)

Signed-off-by: Wirut Getbamrung wgetbumr@celestica.com
